File: WPRK-iOS/WPRK-iOS/Domain/Data/EpisodeDTO/EpisodesDTO.swift
//
//  EpisodesDTO.swift
//  WPRK-iOS
//
//  Created by Mwai Banda on 6/25/22.
//

import Foundation

// MARK: - Episodes
struct EpisodesDTO: Codable {
    let collection: [EpisodeDTO]
    let meta: EpisodeMeta
    enum CodingKeys: String, CodingKey {
        case collection = "data"
        case meta
    }
}

File: WPRKSDK/src/commonMain/kotlin/com/mwaibanda/wprksdk/data/episodeDTO/EpisodesDTO.kt
package com.mwaibanda.wprksdk.data.episodeDTO

import com.mwaibanda.wprksdk.data.metadataDTO.MetadataDTO
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class EpisodesDTO(
    @SerialName("data")
    val episodes: List<EpisodeDTO>,
    val `meta`: MetadataDTO? = null
)